internal/app/app.go

```diff
@@ -181,8 +199,7 @@ func NewDefaultStorage(cfg config.NeutronQueryRelayerConfig, logger *zap.Logger)
 
 func loadChains(
```

Comment:
I believe it's time to split this func to two separate ones: `loadNeutronChain` and `loadTargetChain`:
1. it's quite smol;
2. I personally don't like when a func returns two (or, good lord, more) values of the same type. yes, the return definition contains names that helps to tell them apart, but who reads it?
3. it's got a new parameter `keyName string` that is applicable only for the neutron chain, and from the outside it's not clear what keyname this func expects. so it's either to rename it to `neutronKeyName` (which sucks imo) or split the func (which I fancy)

Reply:
Done, didn't like it either